src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import './index.css';
+
+
+ReactDOM.render(
+  <React.StrictMode>
```

Comment:
I would remove React.StrictMode. I can keep reloading your app and causing your problems. 

```
ReactDOM.render(
    <App />,
  document.getElementById('root')
);
```